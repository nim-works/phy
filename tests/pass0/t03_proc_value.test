discard """
  output: "(Done 1)"
"""
(TypeDefs
  (Int 8)
  (ProcTy (Type 0))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 1) 0 (Locals)
    (List
      (Block (Params)
        (Return (ProcVal 0))
      )
    )
  )
)
