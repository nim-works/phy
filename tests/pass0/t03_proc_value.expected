.type t0 (Int)
.type t1 (Proc t0)
.start t1 p0
.local lo0 int
  LdImmInt 1
  Ret
.end
